.husky/.gitignore

```diff
@@ -1 +0,0 @@
-_
```

Comment:
Is the deletetion of husky files here intentional?

Reply:
No.  I'll revert that.